### ISSUE TITLE
Possibility to hide "Welcome" Page

### ISSUE THREAD

**Issue by BaronNerdington** [tus/tusd#473]
**Is your feature request related to a problem? Please describe.**
I don't want to expose too much information about my TUS instance, e.g. the accurate version i am using and especially the metrics URL.  I want to keep metrics accessible behind an URL of my choice, but since the accurate path is displayed on the welcome page, there is no way to make a "hidden" url out of the box.

I could hide TUS behind a reverse proxy and prohibit GET requests on the `/` url, but that seems like a very big hammer for a very small nail. 

**Describe the solution you'd like**
Introduce a flag e.g. `--hide-welcome-page` like the `--expose-metrics` flag. This would make it possibe to use an obscure URL to make it harder to find the metrics URL without completely disabling metrics.

**Describe alternatives you've considered**
No other solution.

**Can you provide help with implementing this feature?**
Sadly not.

**Additional context**
No context, but a huge thanks for this great piece of software!

**Comment by Acconut** [comment]
Yes, I can see that this would be helpful. However, since this is a relatively low-priority task, I doubt we will work on this soon, unless someone can help.
